src/discordtipbot/discordbot.cr

```diff
@@ -122,10 +122,52 @@ class DiscordBot
     end
   end
 
+  private def split(msg : String, spliterator : String, max_length : Int32)
+    msgs = Array(String).new
+
+    # Break up the message into parts which are less than max_length characters each
+    while (msg.size > max_length)
+      # Find the final space before the max_length
+      pos = msg[0, max_length].rindex(spliterator)
+
+      # If there was no space break at the max_length
+      if (pos.nil?)
```

Comment:
Please remove the parentheses, and consider using `unless foo` instead of `if foo.nil?`. 

Secondly, consider just replacing this and the above lines with:
```cr
pos = msg[0, max_length].rindex(spliterator) || max_length
```